src/test/java/com/google/javascript/clutz/types.d.ts

```diff
@@ -6,7 +6,13 @@ declare namespace ಠ_ಠ.clutz_internal.types {
   var e : any [] ;
   var f : ( ( ) => string ) [] ;
   var functionAndUnion : (a : number , b : any ) => any ;
+  /**
+   * marked const to appear in `compiler.getTopScope().getAllSymbols()`
+   */
   var inferrednum : number ;
```

Comment:
My mental model is that `// ...` is an implementation comment, and we should only emit documentation comments.